### PR TITLE
updating Project.toml for registry

### PR DESCRIPTION
**Pull request by kd-iqvia**
Current version didn't follow semantic versioning so updating it to the initial version for registry.

Thank you for your contribution. You're a :star: already!

Before submitting this PR, please have a look at the below checklist so that we know more about your PR.
Please also reference any relevant issues from the issues page if this PR is intended to address one of those.

## What does this PR do?

- [x] Describe what your pull request does:

- updates `version` for first registry

## Checklist

- [x] Your code builds clean without any errors or warnings

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name = "TreeParzen"
 uuid = "eb66a70c-a255-11e9-03ea-7ba6b2f22006"
 authors = ["iqml <https://github.com/iqml>"]
-version = "0.1.44"
+version = "0.1"
```
Comment (yalwan-iqvia) [inline Project.toml:4]: Should be 0.1.0 I believe, even if 0.1 works correctly, better to be explicit

Comment (kd-iqvia) [reply Project.toml]: a41e329 resolves

File: Project.toml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 name = "TreeParzen"
 uuid = "eb66a70c-a255-11e9-03ea-7ba6b2f22006"
 authors = ["iqml <https://github.com/iqml>"]
```
Comment (yalwan-iqvia) [inline Project.toml:3]: Wanna update author while you're here?

Comment (yalwan-iqvia) [reply Project.toml:3]: Actually my bad, I thought the link wasn't valid but it is

Comment (kd-iqvia) [reply Project.toml:3]: I think it's fine for now. The link is valid for the `iqml` and we might just extend the list at some point  :)

**Pull request merged**